Description: Use SOURCE_DATE_EPOCH for copyright year to make build reproducible
Author: Ondřej Nový <novy@ondrej.org>
Forwarded: not-needed
Last-Update: 2016-04-23

Index: stevedore/doc/source/conf.py
===================================================================
--- stevedore.orig/doc/source/conf.py
+++ stevedore/doc/source/conf.py
@@ -12,6 +12,7 @@
 # serve to show the default.
 
 import datetime
+import os
 import subprocess
 
 # If extensions (or modules to document with autodoc) are in another directory,
@@ -49,7 +50,11 @@ master_doc = 'index'
 
 # General information about the project.
 project = u'stevedore'
-copyright = u'%s, DreamHost' % datetime.date.today().year
+if os.environ.has_key('SOURCE_DATE_EPOCH'):
+    this_year = datetime.datetime.utcfromtimestamp(float(os.environ.get('SOURCE_DATE_EPOCH'))).year
+else:
+    this_year = datetime.date.today().year
+copyright = u'%s, DreamHost' % this_year
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
